workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -214,33 +169,37 @@ async def get_bpolys_from_db(
             'features', json_agg(
                 json_build_object(
                     'type',       'Feature',
-                    'id',         ogc_fid,
+                    'id',         {fid_field},
                     'geometry',   public.ST_AsGeoJSON(geom)::json,
                     'properties', json_build_object(
-                        -- list of fields
-                        'fid', ogc_fid
                     )
                 )
             )
         )
-        FROM {0}
-        WHERE ogc_fid = $1
+        FROM {dataset}
+        WHERE {fid_field} = $1
```

Comment:
Fortunately, `asyncpg` prevents the most obvious threat. Nevertheless, I would be very careful with taking input directly from the user input to a database query without sanity-checking them. Since this doesn't change in this PR, I would advise to open a high-prio issue for this.

Reply:
In Commit e89d9e8 I made to sanity check more explicit by using functions and also avoid code duplication. Also I added missing sanity checks in api.py.

Reply:
Hmm, with the last reviewed version, I got the asyncpg error that I can't execute multiple operations in one query. So somehow I got through to asyncpg. Now I get the `ValueError` :)